pkg/controller/azure/cache/claim.go

```diff
@@ -39,22 +39,36 @@ type RedisClaimController struct{}
 
 // SetupWithManager adds a controller that reconciles RedisCluster resource claims.
 func (c *RedisClaimController) SetupWithManager(mgr ctrl.Manager) error {
+	name := strings.ToLower(fmt.Sprintf("%s.%s.%s",
+		cachev1alpha1.RedisClusterKind,
+		v1alpha2.RedisKind,
+		v1alpha2.Group))
+
 	r := resource.NewClaimReconciler(mgr,
 		resource.ClaimKind(cachev1alpha1.RedisClusterGroupVersionKind),
-		resource.ClassKinds{Portable: cachev1alpha1.RedisClusterGroupVersionKind, NonPortable: v1alpha2.RedisClassGroupVersionKind},
+		resource.ClassKinds{
+			Portable:    cachev1alpha1.RedisClusterGroupVersionKind,
```

Comment:
Finally found one :) should be `cachev1alpha1.RedisClusterClassGroupVersionKind`